lib/manageiq/appliance_console/timezone_configuration.rb

```diff
@@ -8,8 +8,9 @@ class TimezoneConfiguration
     attr_reader   :current_timzone
     attr_accessor :new_timezone
 
-    def initialize(region_timezone_string)
+    def initialize(region_timezone_string, region_new_timezone_string = "")
```

Comment:
Instead of adding a parameter here, couldn't the cli just set the `new_timzone` using the attr_accessor?

Then in a follow up maybe we can remove the current timezone from the parameter here because it doesn't seem to be used.